compiler/src/iree/compiler/Codegen/LLVMCPU/ConvertToLLVM.cpp

```diff
@@ -465,6 +509,11 @@ void ConvertToLLVMPass::runOnOperation() {
   }
   tosa::populateTosaRescaleToArithConversionPatterns(&patterns, use32BitImpl);
 
+  // Make sure we expand any `arith.mulsi_extended` before going to the LLVM
+  // dialect.
+  if (use32BitImpl)
+    patterns.add<ExpandMulSIExtended>(patterns.getContext(), /*benefit=*/1024);
+
```

Comment:
IREE style nit: wrap with braces.

```suggestion
  if (use32BitImpl) {
    patterns.add<ExpandMulSIExtended>(patterns.getContext(), /*benefit=*/1024);
  }
```